std/conv.d

```diff
@@ -5076,6 +5079,75 @@ if (T.length > 0) { return textImpl!dstring(args); }
     assert(text(dg4) == "bool delegate(bool, int) @trusted");
 }
 
+/// Convenience functions for writing arguments to an output range as text.
+void writeText(Sink, T...)(ref Sink sink, T args)
+if (isOutputRange!(Sink, char) && T.length > 0)
+{
+    sink.writeTextImpl!string(args);
+}
+
+/// ditto
+void writeWText(Sink, T...)(ref Sink sink, T args)
+if (isOutputRange!(Sink, wchar) && T.length > 0)
+{
+    sink.writeTextImpl!wstring(args);
+}
+
+/// ditto
+void writeDText(Sink, T...)(ref Sink sink, T args)
+if (isOutputRange!(Sink, dchar) && T.length > 0)
+{
+    sink.writeTextImpl!dstring(args);
+}
+
+///
+@safe unittest
+{
+    import std.array : appender;
+
+    auto output = appender!string();
+    output.writeText("The answer is ", 42);
+
+    assert(output.data == "The answer is 42");
+}
+
+///
+@safe unittest
+{
+    import std.array : appender;
+
+    const color = "red";
+    auto output = appender!string();
+    output.writeText(i"My favorite color is $(color)");
```

Comment:
Looks like dscanner doesn't handle istrings yet?
```
Running DScanner
generated/dscanner-5a53c538d0aa832f03840840271b6631fbbfc53d/dsc --config .dscanner.ini -I ../dmd/druntime/src/ --styleCheck etc std -I.
std/conv.d(5121:23)[error]: Expected `)` instead of stringLiteral `"My favorite color is $(color)"`
std/conv.d(5121:55)[warn]: Empty declaration
```

Reply:
Yes, this is blocked by https://github.com/dlang-community/D-Scanner/pull/965

Reply:
This PR has since been merged. Can we get this over the finish line?

Reply:
Still needs https://github.com/dlang/phobos/pull/10720